Lib/_colorize.py

```diff
@@ -218,6 +229,7 @@ class Theme:
     syntax: Syntax = field(default_factory=Syntax)
     traceback: Traceback = field(default_factory=Traceback)
     unittest: Unittest = field(default_factory=Unittest)
+    difflib: Difflib = field(default_factory=Difflib)
```

Comment:
And here:
```suggestion
    difflib: Difflib = field(default_factory=Difflib)
    syntax: Syntax = field(default_factory=Syntax)
    traceback: Traceback = field(default_factory=Traceback)
    unittest: Unittest = field(default_factory=Unittest)
```

Reply:
I'm very slightly concerned about sorting when the dataclasses aren't `kw_only`.

Should I update the dataclass decorator to include `kw_only=True` and then sort? Or is that out of scope of this PR?

Reply:
:+1: Done